source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -50,8 +50,10 @@ export abstract class Dialog {
             this._overlayElement.className = "acd-dialog-overlay";
             this._overlayElement.onclick = (e) => {
                 // clicks on the overlay window should dismiss the dialog
-                this.preventDefaultAction && this.close();
-            }
+                if (this.preventLightDismissal) {
```

Comment:
I'm not getting this - closing the dialog when clicking on the overlay is what's called "light dismissal" (i.e. clicking outside the dialog closes it.) Now you only do it if the `preventLightDismissal` flag is true, that can't be right?

Reply:
@dclaux  Yes, this is misleading. Kept the default value of preventLightDismissal as `false`.